guides/models/creating-updating-and-deleting-records.md

```diff
@@ -1,71 +1,59 @@
-## Creating Records
+## Création de _Records_
 
-You can create records by calling the
-[`createRecord()`](https://api.emberjs.com/ember-data/release/classes/Store/methods/createRecord?anchor=createRecord)
-method on the store.
+Vous pouvez créer des _records_ (enregistrements) en appelant la méthode
+[`createRecord()`](https://api.emberjs.com/ember-data/release/classes/Store/methods/createRecord?anchor=createRecord) sur le _store_ (magasin, réserve).
 
 ```javascript
 this.store.createRecord('post', {
   title: 'Rails is Omakase',
-  body: 'Lorem ipsum'
+  body: 'Lorem ipsum',
 });
 ```
 
-To make `store` available, you can [inject the `store` service](../#toc_injecting-the-store).
+Pour rendre disponible le _store_, vous pouvez [injecter le service _`store`_](../#toc_injecting-the-store).
 
-## Updating Records
+## Mettre à jour des _Records_
 
-Making changes to EmberData records is as simple as setting the attribute you
-want to change:
+Faire des modifications à des _records_ EmberData est aussi simple que de mettre à jour l'attribut que vous souhaitez modifier&nbsp;:
 
 ```javascript
-this.store.findRecord('post', 1).then(function(post) {
-  // ...after the record has loaded
-  post.title = 'A new post';
+this.store.findRecord('post', 1).then(function (post) {
+  // ...après que le _record_ soit chargé
```

Comment:
That one is very subtle ;) 

Avant que + subjonctif (le complément est une éventualité, on ne sait pas encore si ça arrivera ou pas)
Après que + indicatif (l'action s'est déjà produite, ce n'est plus une éventualité)

```suggestion
  // ...après que le _record_ est chargé
```